assets/js/blocks/product-query/constants.ts

```diff
@@ -78,21 +78,15 @@ export const QUERY_DEFAULT_ATTRIBUTES: QueryBlockAttributes = {
 export const INNER_BLOCKS_TEMPLATE: InnerBlockTemplate[] = [
 	[
 		'core/post-template',
-		{ __woocommerceNamespace: PRODUCT_TEMPLATE_ID },
+		{
+			__woocommerceNamespace: PRODUCT_TEMPLATE_ID,
+			/**
+			 * This class is used to add default styles for inner blocks.
+			 */
+			className: 'products-block-post-template',
+		},
 		[
-			[
-				'woocommerce/product-image',
-				{
-					style: {
-						spacing: {
-							margin: {
-								bottom: '0.75rem',
-								top: '0',
-							},
-						},
-					},
-				},
-			],
+			[ 'woocommerce/product-image' ],
 			[
 				'core/post-title',
```

Comment:
Just to confirm, this block still has styles because it's a variation and we don't want the styling hierarchy to disrupt the default margins, is that correct?

Reply:
Yes, That's correct 🙂

Reply:
:+1: